<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="3" passed="1" failed="2" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2019-12-13T08:25:46Z" name="AlertsTestSuite" finished-at="2019-12-13T08:26:22Z" duration-ms="35829">
    <groups>
    </groups>
    <test started-at="2019-12-13T08:25:46Z" name="AlertsTests" finished-at="2019-12-13T08:26:22Z" duration-ms="35829">
      <class name="com.james.training.selenium_project.uielementspagetests.AlertsSoftAssertsTests">
        <test-method is-config="true" signature="setUp(java.lang.String, org.testng.ITestContext)[pri:0, instance:com.james.training.selenium_project.uielementspagetests.AlertsSoftAssertsTests@1ebd319f]" started-at="2019-12-13T08:25:46Z" name="setUp" finished-at="2019-12-13T08:25:54Z" duration-ms="7709" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[org.testng.TestRunner@7d0614f]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="jsAlertTest()[pri:0, instance:com.james.training.selenium_project.uielementspagetests.AlertsSoftAssertsTests@1ebd319f]" started-at="2019-12-13T08:25:54Z" name="jsAlertTest" finished-at="2019-12-13T08:25:57Z" duration-ms="3639" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The following asserts failed:Alert message is not expected. Should be 'I am a JS Alert', but it is 'I am a JS Alert[Fail]' expected [true] but found [false],result is not expected. Should be 'You successfuly clicked an alert', but it is 'You successfuly clicked an alert[Fail]' expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The following asserts failed:Alert message is not expected. Should be 'I am a JS Alert', but it is 'I am a JS Alert[Fail]' expected [true] but found [false],result is not expected. Should be 'You successfuly clicked an alert', but it is 'You successfuly clicked an alert[Fail]' expected [true] but found [false]at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)at com.james.training.selenium_project.uielementspagetests.AlertsSoftAssertsTests.jsAlertTest(AlertsSoftAssertsTests.java:52)at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)at java.base/java.lang.reflect.Method.invoke(Method.java:566)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.Invoker.invokeMethod(Invoker.java:571)at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:707)at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:979)at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)at org.testng.TestRunner.privateRun(TestRunner.java:648)at org.testng.TestRunner.run(TestRunner.java:505)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- jsAlertTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.james.training.selenium_project.uielementspagetests.AlertsSoftAssertsTests@1ebd319f]" started-at="2019-12-13T08:25:57Z" name="tearDown" finished-at="2019-12-13T08:25:59Z" duration-ms="1299" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setUp(java.lang.String, org.testng.ITestContext)[pri:0, instance:com.james.training.selenium_project.uielementspagetests.AlertsSoftAssertsTests@1ebd319f]" started-at="2019-12-13T08:25:59Z" name="setUp" finished-at="2019-12-13T08:26:05Z" duration-ms="6535" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[org.testng.TestRunner@7d0614f]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="jsDismissTest()[pri:0, instance:com.james.training.selenium_project.uielementspagetests.AlertsSoftAssertsTests@1ebd319f]" started-at="2019-12-13T08:26:05Z" name="jsDismissTest" finished-at="2019-12-13T08:26:09Z" duration-ms="3581" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The following asserts failed:Alert message is not expected. Should be 'I am a JS Confirm', but it is 'I am a JS Confirm[Fail]' expected [true] but found [false],result is not expected. Should be 'You clicked: Cancel', but it is 'You clicked: Cancel[Fail]' expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The following asserts failed:Alert message is not expected. Should be 'I am a JS Confirm', but it is 'I am a JS Confirm[Fail]' expected [true] but found [false],result is not expected. Should be 'You clicked: Cancel', but it is 'You clicked: Cancel[Fail]' expected [true] but found [false]at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)at com.james.training.selenium_project.uielementspagetests.AlertsSoftAssertsTests.jsDismissTest(AlertsSoftAssertsTests.java:88)at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)at java.base/java.lang.reflect.Method.invoke(Method.java:566)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.Invoker.invokeMethod(Invoker.java:571)at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:707)at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:979)at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)at org.testng.TestRunner.privateRun(TestRunner.java:648)at org.testng.TestRunner.run(TestRunner.java:505)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- jsDismissTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.james.training.selenium_project.uielementspagetests.AlertsSoftAssertsTests@1ebd319f]" started-at="2019-12-13T08:26:09Z" name="tearDown" finished-at="2019-12-13T08:26:10Z" duration-ms="1026" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setUp(java.lang.String, org.testng.ITestContext)[pri:0, instance:com.james.training.selenium_project.uielementspagetests.AlertsSoftAssertsTests@1ebd319f]" started-at="2019-12-13T08:26:10Z" name="setUp" finished-at="2019-12-13T08:26:16Z" duration-ms="6475" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[org.testng.TestRunner@7d0614f]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="jsPromptTest()[pri:0, instance:com.james.training.selenium_project.uielementspagetests.AlertsSoftAssertsTests@1ebd319f]" started-at="2019-12-13T08:26:16Z" name="jsPromptTest" finished-at="2019-12-13T08:26:21Z" duration-ms="4555" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- jsPromptTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.james.training.selenium_project.uielementspagetests.AlertsSoftAssertsTests@1ebd319f]" started-at="2019-12-13T08:26:21Z" name="tearDown" finished-at="2019-12-13T08:26:22Z" duration-ms="982" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.james.training.selenium_project.uielementspagetests.AlertsSoftAssertsTests -->
    </test> <!-- AlertsTests -->
  </suite> <!-- AlertsTestSuite -->
</testng-results>
